RPA.MIT.Notification.Function/Validation/ValidateMessage.cs

```diff
@@ -30,7 +33,8 @@ public static class ValidateMessage
             ""Data"",
             ""Id"",
             ""Scheme"",
-            ""EmailRecipient""
+            ""ApproverEmail"",
```

Comment:
Same thing here. Should just be 'EmailRecipient'

Reply:
Reverted the changes